src/app/wallet/v2/views/asset_list.nim

```diff
@@ -55,16 +66,33 @@ QtObject:
     of AssetRoles.Description: result = newQVariant(asset.description)
     of AssetRoles.Permalink: result = newQVariant(asset.permalink)
     of AssetRoles.ImageUrl: result = newQVariant(asset.imageUrl)
+    of AssetRoles.BackgroundColor: result = newQVariant(if (asset.backgroundColor == ""): "transparent" else: ("#" & asset.backgroundColor))
+    of AssetRoles.Properties:
+        self.propertiesList.traits = @[]
```

Comment:
Does it mean that each time you access the attributes properties, it also mutate the asset_list view?
If I understand that correctly this is probably wrong that a reader is also mutating internal element of the view right?

If I am correct in my understanding, what about creating an single asset view: which would be set explicitly when viewing a specific asset

wdyt?

Reply:
Hey @alaibe It does not mutate the whole asset_list at all. I have only created the properties, ranking, stats list to be able to pass the data in the OpenseaAsset. If you see the line -
        self.propertiesList.traits = asset.properties
this is only used to update the properties list with the data in the asset. This is done to be able to pass it on to QML as a abstractListItemModel. 

Asset list contains the list of all assets, each asset has lists of properties , ranking and stats and I simply use the 
propertiesList*: TraitsList
     rankingList*: TraitsList
     statsList*: TraitsList
to pass then information already stored in the asset to the UI side.

Reply:
Addressed all the other review comments :)

Reply:
Gotcha thanks for clarification. I think the only possible problem of the solution is that it cannot shows traits of 2 assets at the same time

But as this is not part of the design, I guess it's ok!
Sorry for delay